tests/test-repo-finder-mount.c

```diff
@@ -464,7 +464,7 @@ test_repo_finder_mount_well_known (Fixture       *fixture,
   g_autoptr(GError) error = NULL;
   g_autoptr(GList) mounts = NULL;  /* (element-type OstreeMockMount)  */
   g_autoptr(GMount) mount = NULL;
-  glnx_fd_close int repos = -1;
+  glnx_autofd int repos = -1;
```

Comment:
After looking at the full diff, the obvious thing that comes to mind is; would it make sense to just absorb the `int` into the macro? Unlike e.g. `g_autofree`, there's only a single type we would ever use this with, right? Just a thought! 💡